editor/plugins/script_text_editor.cpp

```diff
@@ -1854,23 +1854,13 @@ void ScriptTextEditor::_enable_code_editor() {
 
 	add_child(color_panel);
 
-	color_picker = memnew(ColorPicker);
-	color_picker->set_deferred_mode(true);
+	color_picker = memnew(ColorPickerButton);
+	color_picker->get_picker()->set_deferred_mode(true);
 	color_picker->connect("color_changed", callable_mp(this, &ScriptTextEditor::_color_changed));
```

Comment:
This picker appears when you right-click a Color in the script editor.
![godot windows tools 64_EzFengNOeV](https://user-images.githubusercontent.com/2223172/176881894-2b51b580-b506-4231-ac77-e2eab4c25a58.gif)
It must be ColorPicker to appear correctly. Not sure why you changed the type if the method expects ColorPicker.